cli/command/stack/list_test.go

```diff
@@ -14,6 +15,10 @@ import (
 	"gotest.tools/golden"
 )
 
+var (
+	orchestrator = commonOptions{orchestrator: command.OrchestratorSwarm}
```

Comment:
`orchestrator` is actually a `commonOptions`?

Reply:
Yes I know, I was lazy... It was a first attempt with orchestrator, didn't want to rename 😅